hikari/impl/interaction_server.py

```diff
@@ -86,26 +86,29 @@
 _USER_AGENT_KEY: typing.Final[str] = "User-Agent"
 _APPLICATION_OCTET_STREAM: typing.Final[str] = "application/octet-stream"
 _JSON_CONTENT_TYPE: typing.Final[str] = "application/json"
-_JSON_TYPE_WITH_CHARSET: typing.Final[str] = f"{_JSON_CONTENT_TYPE}; charset={_UTF_8_CHARSET}"
 _TEXT_CONTENT_TYPE: typing.Final[str] = "text/plain"
-_TEXT_TYPE_WITH_CHARSET: typing.Final[str] = f"{_TEXT_CONTENT_TYPE}; charset={_UTF_8_CHARSET}"
 
 
 class _Response:
-    __slots__: typing.Sequence[str] = ("_content_type", "_files", "_payload", "_status_code")
+    __slots__: typing.Sequence[str] = ("_content_type", "_charset", "_files", "_payload", "_status_code")
 
     def __init__(
         self,
         status_code: int,
         payload: typing.Optional[bytes] = None,
         *,
         content_type: typing.Optional[str] = None,
+        charset: typing.Optional[str] = None,
```

Comment:
Since this is seemingly never actually provided in the real world, would make more sense to just have it be static for this implementation and have the charset property always return _UTF_8_CHARSET

Reply:
That's fair, updated